gwcli/client.py

```diff
@@ -2,13 +2,14 @@
 
 from gwcli.node import UIGroup, UINode
 
-from gwcli.utils import response_message, valid_iqn, APIRequest
```

Comment:
Nit: can we delete the ```valid_iqn``` function if it's no longer used?

Reply:
It's still used in valid_client() to check if the name is in correct format.

I was thinking I could just remove that check since it is normalized before valid_client() is called but I was thinking maybe other callers might want to use valid_client to check still.

Reply:
Can that method be updated to use the same ```normalize_wwn``` method?

Reply:
Yeah, I see what you mean now. Done.